cli/args/flags.rs

```diff
@@ -4706,6 +4707,7 @@ fn add_parse(
 ) -> clap::error::Result<()> {
   allow_scripts_arg_parse(flags, matches)?;
   lock_args_parse(flags, matches);
+  config_args_parse(flags, matches);
```

Comment:
I think you might need to manually parse it here without it taking into account no-config